src/BlockEntities/CommandBlockEntity.cpp

```diff
@@ -178,6 +178,12 @@ void cCommandBlockEntity::Execute()
 		}
 	} CmdBlockOutCb(this);
 
+	// Remove leading slash if one exists, since console commands don't use them
+	if (m_Command[0] == '/')
```

Comment:
Can this codepath be taken even when `m_Command` is empty? Then this would fail.